content/applications/websites/ecommerce/checkout_payment_shipping/shipping.rst

```diff
@@ -1,70 +1,69 @@
-================
-Shipping methods
-================
+========
+Shipping
+========
 
-Depending on your shipping strategy, you have the choice to either use your :ref:`own shipping
-methods <ecommerce-own-shipping>`, or use an integration with an :ref:`existing shipping provider
-<ecommerce-shipping-providers>`.
+When shopping online, customers can choose their shipping method from the options offered at the
+time of :doc:`checkout <../checkout_payment_shipping/checkout>`.
 
-.. _ecommerce-own-shipping:
+You can integrate with :ref:`external providers <ecommerce/shipping/external-provider>` to handle
+deliveries, fetch real-time rates, generate labels, and track shipments, or offer
+:ref:`custom shipping options <ecommerce/shipping/custom-method>` like flat-rate shipping, free
+shipping, or :doc:`Sendcloud <../../../inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping>`
+carriers.
```

Comment:
(if we add a Pick up in store section (see my other comments), it might be worth it to mention it as well.